Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -7,7 +7,7 @@
 # vending machines. An extra hundred or two profit margin should be fine.
 
 - type: cargoProduct
-  id: CrateVendingMachineRestockBooze
+  id: CrateVendingMachinRestockBooze
```

Comment:
???

Reply:
Oh, I think my cats walked on my keyboard when I wasn't looking. When I first submitted, it was accidentally changed to CrateVendingMachineRestockBoeadtrefze or something like that. 😹. So, I had to commit a fix after I noticed.

Reply:
Oh, and I just noticed it's still missing the damn e.... I'm at work now and can't fix at the moment.

Reply:
Had a min. Should be fixed now.